lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -393,7 +393,13 @@ def insert_at_position(position)
         end
 
         def update_positions
-          old_position = send("#{position_column}_was") || bottom_position_in_list + 1
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
Makes sense. Let's just leave it at a direct comparison for now, and we can come back to it later.  So no change needed here, just the other minor one and this is good to go.